frontend/package.json

```diff
@@ -11,11 +11,17 @@
     "test:unit": "vue-cli-service test:unit"
   },
   "dependencies": {
+    "axios": "^0.18.0",
+    "bcrypt": "^3.0.6",
+    "body-parser": "^1.19.0",
     "core-js": "^2.6.5",
+    "jsonwebtoken": "^8.5.1",
     "material-design-icons-iconfont": "^3.0.3",
     "register-service-worker": "^1.6.2",
     "roboto-fontface": "*",
+    "sqlite3": "^4.0.8",
```

Comment:
What do you need sqlite3 for? This is gonna be run in a browser.

Reply:
will remove